pkg/cluster/internal/kubeadm/config.go

```diff
@@ -272,8 +282,8 @@ evictionHard:
   nodefs.inodesFree: "0%"
   imagefs.available: "0%"
 {{if .FeatureGates}}featureGates:
-{{ range $key := .SortedFeatureGateKeys }}
-  "{{ $key }}": {{ index $.FeatureGates $key }}
+{{ range $index, $gate := .SortedFeatureGates }}
+  "{{ (StructuralData $gate.Name) }}": {{ $gate.Value }}
```

Comment:
StructuralData is a special function to tell safetext that this is *supposed* to result in a dynamic input-controlled key and that's expected.